### DISCUSSION TITLE
Bootstrap or w3.css or some other alternative?

### DISCUSSION THREAD

**Discussion by coderGtm** [coderGtm/ojaswat-web-page#2] [Polls]
As we aim to make the webpage responsive, we are likely to use a framework for that. Two frameworks which I know are great are Bootstrap and w3.css. There might be other frameworks better so please list them in the poll. What shall we choose? Also write your reason so we can discuss.

**Comment by coderGtm** [comment]
I personally prefer w3.css as this is just a small webpage so the **heaviness** of Bootstrap is not required.

**Reply by andrewbabu** [reply]
If you are comfortable with w3.css then we can try it .I am new to it

**Reply by coderGtm** [reply]
I am comfortable with both. I just think that w3.css is more suited for this task.

**Reply by andrewbabu** [reply]
ok the let's continue with w3.css

**Reply by coderGtm** [reply]
If you are convinced then cast your vote so we can close this

**Reply by coderGtm** [reply]
Ok so we're going with w3.css

**Comment by coderGtm** [comment]
Seems as if BS5 is better. The problem with heaviness can be tackled by using CDN. And seeing the requirements, BS5 is better suited and faster.

**Reply by coderGtm** [reply]
Why Use Bootstrap?
Advantages of Bootstrap:

- Responsive features: Bootstrap's responsive CSS adjusts to phones, tablets, and desktops
- Mobile-first approach: In Bootstrap, mobile-first styles are part of the core framework
- Browser compatibility: Bootstrap 5 is compatible with all modern browsers (Chrome, Firefox, Edge, Safari, and Opera).
